wiki/Guides/Starting_a_Beatmap_Project/fr.md

```diff
@@ -1,36 +1,41 @@
 # Comment démarrer un projet de beatmap
 
-*[Veuillez préciser le type de projet (Discographie, Album, Collab) par LuigiHann](https://osu.ppy.sh/community/forums/topics/12193)*
-
-Note : Cet article concerne la section du forum [Beatmap Projects](https://osu.ppy.sh/community/forums/53) et la manière de l'aborder.
+*Remarque: Cet article concerne la section du forum [Beatmap Projects](https://osu.ppy.sh/community/forums/53) et la manière de l'aborder. Assurez-vous de lire les [règles du forum](https://osu.ppy.sh/community/forums/topics/453937) avant de poster vos messages.*
```

Comment:
```suggestion
*Remarque : Cet article concerne la section du forum [Beatmap Projects](https://osu.ppy.sh/community/forums/53) et la manière de l'aborder. Assurez-vous de lire les [règles du forum](https://osu.ppy.sh/community/forums/topics/453937) avant de poster vos messages.*
```

Reply:
Why need extra whitespace?

Reply:
because french. check any other french article - punctuation rules are different

Reply:
Ah I see why every marks should require space in before.

Reply:
In French you always put a space in this way, it is often forgotten in translations because Visual Studio Code does not correct these mistakes, despite a corrector.
But I confess myself I don't understand why we do it like that, but it's the French language that wants that